code/game/objects/items/devices/radio/headset.dm

```diff
@@ -165,6 +165,11 @@
 			to_chat(user, SPAN_NOTICE("This headset doesn't have any encryption keys!  How useless..."))
 
 	if(istype(W, /obj/item/device/encryptionkey/))
+		for (var/obj/item/device/encryptionkey/key in keys)
+			if (key.type == W.type)
+				to_chat(user, SPAN_NOTICE("There is already the same type of encryption key in the headset!"))
```

Comment:
seems off, feel like this would sound better, untested btw. 
```suggestion
				to_chat(user, SPAN_NOTICE("\The [W.name] is already installed on this device."))
```

Reply:
Yeah, thanks for the correction! I tested this change and its better this way.